cmd/neofs-node/tree.go

```diff
@@ -81,7 +81,7 @@ func syncTrees(ctx context.Context, treeSvc *tree.Service, cnrCli *containerClie
 
 	for _, id := range ids {
 		for _, tID := range wellKnownTrees {
-			err = treeSvc.Synchronize(ctx, id, tID)
+			err = treeSvc.Synchronize(ctx, id, &tID)
```

Comment:
Taking a pointer to a loop variable can bite us unexpectedly if we ever go async inside of a `Synchronize`. What about `&wellKnownTrees[i]`?

Reply:
good point. changed (it was a temporary change though)